### PR TITLE
Add gitops compatible apps

### PR DESCRIPTION
**Pull request by pranavgaikwad**
This PR clones existing apps in a different directory compatible for deployments through GitOps Pipelines

### CODE REVIEW

File: gitops/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - /spec/template/spec/containers/0/terminationMessagePath
+    - /spec/template/spec/containers/0/terminationMessagePolicy
+    - /spec/template/spec/containers/1/image
+    - /spec/template/spec/containers/1/ports/0/protocol
+    - /spec/template/spec/containers/1/terminationMessagePath
+    - /spec/template/spec/containers/1/terminationMessagePolicy
+    kind: DeploymentConfig
+  project: default
+  source:
+    path: <most_specific_overlay_or_base_for_your_envirogit nment>
```
Comment (alaypatel07) [inline gitops/README.md:50]: typo?
`<most_specific_overlay_or_base_for_your_environment>` assume you meant?

File: gitops/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+```yml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: rocket-chat-local
+  namespace: openshift-gitops
+spec:
+  destination:
+    namespace: <namespace_in_which_you_want_to_deploy_this_app>
+    server: <target_cluster_to_deploy_to>
```
Comment (alaypatel07) [inline gitops/README.md:36]: does this URL has to be the same as the cluster argocd operator is installed in?

Comment (pranavgaikwad) [reply gitops/README.md:36]: @alaypatel07 you can add multiple clusters to ArgoCD instance. So you can manage deployments to remote clusters too. Doesn't have to be the cluster on which ArgoCD is deployed

Comment (alaypatel07) [reply gitops/README.md:36]: > you can add multiple clusters to ArgoCD instance.

Very interesting. Thanks!

**Comment by pranavgaikwad** [comment]
Tested all the apps, merging! Thanks for the review.

**Pull request merged**